app/views/leader_sessions/sign_in.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1>Sign in</h1>
+
+<form action="/verify_credentials" method="post">
```

Comment:
Looks like this action still needs updating?

Reply:
Nice catch!